frontend/packages/ceph-storage-plugin/src/utils/install.ts

```diff
@@ -48,10 +52,13 @@ export const getTotalDeviceCapacity = (list: Discoveries[]): number =>
 
 export const getAssociatedNodes = (pvs: K8sResourceKind[]): string[] => {
   const nodes = pvs.reduce((res, pv) => {
-    const nodeName = pv?.metadata?.labels?.[HOSTNAME_LABEL_KEY];
```

Comment:
can you tell in which case the HOSTNAME_LABEL_KEY will not be present if =done from localVolumeSet?

Reply:
There is not any case for local volume set.
The bug is for local volume fix  and an improvement over the current implementation where we identify nodes via labels.
See https://github.com/openshift/console/pull/7552/files#r543672246